modules/programs/ssh.nix

```
@@ -316,7 +324,7 @@ in
         map matchBlockStr (
         builtins.attrValues cfg.matchBlocks))}
 
-      Host *
+      Host ${yn cfg.host}
```

```suggestion
      Host ${cfg.host}
```

the `yn` function will attempt to turn a boolean into a `"yes"` or a `"no"`.